actions/imanage/functions/imanage-login/1.0/index.js

```diff
@@ -0,0 +1,33 @@
+import urlSearchParams from "../../utils/url-search-params"
+
+const AUTHURL = "https://cloudimanage.com/auth/oauth2/token"
+
+const imanageLogin = async ({email, password, client_id, client_secret}) => {
+    const formData = new urlSearchParams({
```

Comment:
formData naming seems a bit off. formData is a actually a different class used is some requests: https://developer.mozilla.org/en-US/docs/Web/API/FormData